README.md

```diff
@@ -358,13 +357,13 @@ service studentData on studentServiceListener {
         if (resResult is error) {
             log:printError("Error sending response", err = resResult);
         }
-        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
+        //  The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
```

Comment:
The below function adds tags to be passed as metrics in the message traces. These tags are added to the default system span.